### PR TITLE
Initial deploy of concept-ingester

### PR DESCRIPTION
**Pull request by scott-ace-newton**

### CODE REVIEW

File: concept-ingester-blue@.service [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+ExecStartPre=/bin/bash -c '/usr/bin/docker pull coco/concept-ingester:$DOCKER_APP_VERSION'
+
+ExecStart=/bin/sh -c '\
+  export PORT=8080; \
+  export VULCAN_ADDR=http://%H:8080; \
+  export GROUP_ID=BlueConcepts; \
+  export TOPIC=Concept; \
+  export SERVICES=people-rw-neo4j-blue,organisations-rw-neo4j-blue,memberships-rw-neo4j-blue; \
+  export HOST_HEADER=kafka; \
+  export OFFSET=smallest; \
```
Comment (catalin-ilea) [inline concept-ingester-blue@.service]: Can you please set this to largest?

File: services.yaml [hunk 2]
```diff
@@ -56,6 +56,12 @@ services:
 - name: cms-notifier@.service
   version: v33
   count: 1
+- name: concept-ingester-blue@.service
+  version: v0.0.15
+  count: 1 
+- name: concept-ingester-red@.service
+  version: v0.0.15
+  count: 1
```
Comment (Zapadlo) [inline services.yaml]: Sidekicks?

File: concept-ingester-blue@.service [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+Wants=concept-ingester-blue-sidekick@%i.service
+
+[Service]
+Environment="DOCKER_APP_VERSION=latest"
+TimeoutStartSec=0
+# Change killmode from "control-group" to "none" to let Docker remove work correctly.
+KillMode=none
+ExecStartPre=-/bin/bash -c '/usr/bin/docker kill %p-%i > /dev/null 2>&1'
+ExecStartPre=-/bin/bash -c '/usr/bin/docker rm %p-%i > /dev/null 2>&1'
+ExecStartPre=/bin/bash -c '/usr/bin/docker pull coco/concept-ingester:$DOCKER_APP_VERSION'
```
Comment (Zapadlo) [inline concept-ingester-blue@.service]: Do you mind updating this to be in the following format?

`ExecStartPre=/bin/bash -c 'docker history coco/concept-ingester:$DOCKER_APP_VERSION > /dev/null 2>&1 || docker pull coco/concept-ingester:$DOCKER_APP_VERSION'`

Same for red

Comment (scott-ace-newton) [inline concept-ingester-blue@.service]: sure. can you explain what it is doing please

Comment (Zapadlo) [inline concept-ingester-blue@.service]: Checks if you have the image on disk first, before pulling.

**Pull request closed without merging**